src/go/k8s/go.mod

```diff
@@ -0,0 +1,22 @@
+module github.com/vectorizedio/redpanda/src/go/k8s
+
+go 1.15
+
+require (
+	github.com/go-logr/logr v0.3.0
+	github.com/onsi/ginkgo v1.14.1
+	github.com/onsi/gomega v1.10.2
+	github.com/sirupsen/logrus v1.7.0 // indirect
+	github.com/spf13/viper v1.7.0 // indirect
+	github.com/vectorizedio/redpanda/src/go/rpk v0.0.0-20210115135306-1a769e995810
```

Comment:
Shall we also go with a replace directive for the internal module, since they are versioned together?
`github.com/vectorizedio/redpanda/src/go/rpk v0.0.0 => ../rpk`?

Reply:
Yes. Changed, thank you.